src/neo/Ledger/Blockchain.cs

```diff
@@ -516,6 +450,8 @@ private void Persist(Block block)
                     header_index.Add(block.Hash);
```

Comment:
This should be removed.

Reply:
In `Persist(GenesisBlock);` and `OnImport`, we need it.

Reply:
Then move it to the beginning of `Persist()`, and remove it from L334.

Reply:
done and tested.